src/booking/containers/booking-location/booking-location.component.ts

```diff
@@ -52,6 +52,10 @@ export class BookingLocationComponent implements AfterViewInit, OnInit {
     }
   }
 
+  public selectedLocations(): LocationByEPIMMSModel[] {
+    return this.bookingProcess.location ? [this.bookingProcess.location] : [];
```

Comment:
Do we need this method? Can it have a default value of []
Could init method have the logic?